content/blog/2019/2019-09-28-on-expected-vomiting-time.md

```diff
@@ -0,0 +1,90 @@
++++
+author="Janez Demšar"
+date= '2019-09-28'
+draft= true
+title="On Expected Vomiting Time"
+type="blog"
+thumbImage = "/blog_img/2019-09-28-vomiting-pumpkin.jpg"
+frontPageImage = "/blog_img/2019-09-28-vomiting-pumpkin.jpg"
+categories=["model performance", "confusion matrix"]
```

Comment:
I would also add "education", so we can keep track of all the wonderful teaching materials we have. :)